src/commonmark/en/content/developer/web-api/analytics.md

```diff
@@ -155,8 +155,20 @@ The analytics resource lets you specify a range of query parameters:
 <tr class="even">
 <td>outputIdScheme</td>
 <td>No</td>
-<td>Identifier scheme to use for metadata items the query response, can be identifier, code or attributes.</td>
-<td>UID | CODE |NAME| ATTRIBUTE:&lt;ID&gt;</td>
+<td>Identifier scheme used for metadata items in the query response. It accepts identifier, code or attributes.</td>
+<td>UID | UUID | CODE | NAME | ATTRIBUTE:&lt;ID&gt;</td>
```

Comment:
Should it have both UID and UUID here?

Reply:
@maikelarabori can you confirm both UID and UUID here?

Reply:
Had a look at the code. It is valid to specify both UID or UUID, however, it seems both will refer to UID :) So it's correct according to the code